
＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,884                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15239]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,897                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15252]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,898                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15252]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,898                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15252]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,898                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15253]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,899                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15253]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,899                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15254]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,900                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15254]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,900                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15255]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,900                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15255]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,901                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15255]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,901                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15256]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,902                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15256]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,902                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15257]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,903                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15257]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,903                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15257]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,903                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15258]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,904                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15258]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,904                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15259]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,905                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15259]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,905                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15259]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,905                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15260]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,906                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15260]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,906                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15261]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,907                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15261]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,907                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15261]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,907                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15262]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,908                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15262]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,908                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15263]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,908                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15263]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,909                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15263]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,909                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15264]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,910                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15264]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,910                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15264]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,910                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15265]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,911                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15265]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,911                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15265]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,911                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15266]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,912                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15266]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,912                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15267]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,914                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15269]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,915                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15269]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,915                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15270]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,915                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15270]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,916                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15270]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,916                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15271]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,917                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15271]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,917                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15272]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,918                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15272]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,918                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15272]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,918                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15273]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,919                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15273]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,919                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15274]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,919                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15274]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,920                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15274]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,920                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15275]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,921                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15275]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,921                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15276]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,921                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15276]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,922                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15276]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,922                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15277]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,923                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15277]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,923                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15277]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,923                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15278]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,924                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15278]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,924                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15279]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,924                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15279]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,925                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15279]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,925                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15280]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,926                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15280]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,926                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15280]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,927                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15281]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,927                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15281]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,927                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15282]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,928                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15282]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,928                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15283]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,928                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15283]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,929                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15283]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,929                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15284]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,930                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15284]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,930                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15284]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,930                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15285]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,497                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14852]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,931                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15286]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,931                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15286]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,932                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15286]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,932                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15287]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,933                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15287]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,933                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15287]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,933                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15288]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,934                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15288]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,934                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15289]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,934                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15289]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,935                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15289]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,935                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15290]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,935                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15290]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,936                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15290]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,936                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15291]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,937                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15291]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,937                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15291]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,937                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15292]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,938                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15292]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,938                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15292]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,938                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15293]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,939                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15293]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,939                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15294]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,939                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15294]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,940                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15294]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,940                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15295]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,941                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15295]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,941                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15296]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,941                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15296]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,705                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15060]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26
